paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -470,8 +470,19 @@ internal class CustomerSheetViewModel(
         }
     }
 
-    private fun onItemRemoved(paymentMethod: PaymentMethod) {
+    private fun onItemRemoved(paymentMethodId: String?) {
+        // TODO(samer-stripe): We should guarantee a payment method ID is available with data modeling updates to UI
+        if (paymentMethodId == null) {
+            return
+        }
```

Comment:
Didn't think we should do this check in the UI. Ideally we have better data modeling to not do this at all. Wondering if should keep this here until we do.

Reply:
Will also write tests for these cases if we want to make these nullable for now.